core/controllers/profile.py

```diff
@@ -79,6 +79,7 @@ def get(self, username):
 
         self.values.update({
             'user_bio': user_settings.user_bio,
+            'subject_interests': ", ".join(user_settings.subject_interests),
```

Comment:
Was unsure how to let the frontend take care of the formatting. Until then, I have used utils.get_comma_sep_string_from_list() in profile.py